Rocket.Chat/Controllers/Base/BaseViewController.swift

```diff
@@ -9,6 +9,11 @@
 import UIKit
 
 class BaseViewController: UIViewController {
+    func alert(title: String, message: String) {
+        let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .default, handler: nil))
+        present(alert, animated: true, completion: nil)
+    }
```

Comment:
Nice!!! Maybe we can look for usages in the code and also substitute this in another PR? Also, what do you think about adding this method to an extension of `BaseViewControllerAlerts.swift`?